nautobot/ipam/utils/migrations.py

```diff
@@ -105,36 +143,37 @@ def process_vrfs(apps):
     Returns:
         None
     """
+    Namespace = apps.get_model("ipam", "Namespace")
     VRF = apps.get_model("ipam", "VRF")
+
+    global_ns = Namespace.objects.get(name=GLOBAL_NS)
     vrfs = VRF.objects.all().order_by("name", "rd")
     unique_non_empty_vrfs = vrfs.filter(enforce_unique=True).exclude(ip_addresses__isnull=True, prefixes__isnull=True)
     # At the point in the migration where we iterate through vrfs in global_ns_vrfs, every vrf that
     # has already been processed has been moved to a new namespace. Anything left in the global
     # namespace has yet to be processed which is why we're iterating through this on the second
     # loop.
-    global_ns_vrfs = vrfs.filter(namespace__name=GLOBAL_NS)
+    global_ns_vrfs = vrfs.filter(namespace=global_ns)
 
     # Case 0: VRFs with enforce_unique move to their own Namespace.
-    for vrf in unique_non_empty_vrfs:
+    for vrf in unique_non_empty_vrfs.iterator():
         if "test" not in sys.argv:
             print(f">>> Processing migration for VRF {vrf.name!r}, Namespace {vrf.namespace.name!r}")
         vrf.namespace = create_vrf_namespace(apps, vrf)
         vrf.save()
         vrf.prefixes.update(namespace=vrf.namespace)
-        if "test" not in sys.argv:
```

Comment:
Was this condition removed intentionally?

Reply:
Yes, for symmetry with the below case (line 137/176) that has no similar conditional. I can change it back if desired.